README.md

```diff
@@ -1,2 +1,103 @@
+
 # nora-service
 NORA (https://node-red-google-home.herokuapp.com) backend service deployed in Heroku.
+
+Deployment instructions: (small guide on how to run nora-service on your own Heroku)
+
+Needed:
+ - [Heroku app](https://www.heroku.com/)
+ - [Firebase App](https://console.firebase.google.com/)
+ - [Google Api Console oauth](https://console.developers.google.com/)
+ - [Google Actions](https://console.actions.google.com/)
+
+## HEROKU
+
+- Create a new Account and then a new app. From Overview, add new add-on "Heroku Postgres"
+To deploy, you can either use the heroku cli or connect your github account. (do not deploy anything yet)
+ - Keep this browser tab open
+
+## FIREBASE
+
+ - Create a new Firebase Project and name it accordingly.
+ - Navigate to Project Overview - Project Settings - General and create a new web app.
+ - Navigate to Project Overview - Project Settings - Service Accounts and generate new private key
+ - Navigate to Develop - Authentication - Users - Set up sign-in method - Google - enable - check that Web SDK configuration is filled in.
+ - Keep this browser tab open
+
+## Google Api Console
+
+- Create a Project
+- Enable [HomeGraph API](https://console.developers.google.com/apis/api/homegraph.googleapis.com/)
+- Under 'OAuth 2.0 Client IDs' click your webclient credentials
+- Add under URIs your Heroku url eg https://XXXXXXXXX.herokuapp.com
+- Keep this browser tab open
+
+## Google Actions
+
+- Create a project
+- Choose 'Smart Home'
+- Name your Action
+- Go back to Overview and Setup Account Linking 
+	- Linking Type: OAuth - Authorization Code
+	- Client ID: this is your Client Id from Google API Console ('OAuth 2.0 Client IDs' click your webclient credentials)
+	- Client secret: this is your Client Secret from Google API Console ('OAuth 2.0 Client IDs' click your webclient credentials)
+	- Authorization URL - https://XXXXXXXXX.herokuapp.com/oauth
+	- Token URL - https://XXXXXXXXX.herokuapp.com/oauth/token
+	- Scopes (I added google-home-authcode / app-user because i saw those 2 somewhere in the code but I don't think they are needed) ?? @Andrei please confirm/deny
+- Back in Overview click Add Actions
+- Fulfillment URL - https://XXXXXXXXX.herokuapp.com/smarthome/fulfill
+
+
+## CONFIG.TS
+
+ Pull the repo, and modify [src/config.ts](https://github.com/andrei-tatar/nora-service/blob/master/src/config.ts)
+
+    export const oauthClientId = 'api console - credentials - oauth 2.0 client ids - web client - Client ID';
```

Comment:
The code doesn't need changing. These values are pulled from environment variables which can be set directly in heroku /settings/config vars

![image](https://user-images.githubusercontent.com/6473183/80377036-5d9f7700-88a3-11ea-8a55-c63bb6442268.png)